client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -422,21 +422,18 @@ export class CheckoutThankYou extends React.Component {
 					<AtomicStoreThankYouCard siteId={ this.props.selectedSite.ID } />
 				</Main>
 			);
-		} else if ( wasDotcomPlanPurchased && ( delayedTransferPurchase || this.isNewUser() ) ) {
-			let planProps = {};
-			if ( delayedTransferPurchase ) {
-				planProps = {
-					action: (
-						// eslint-disable-next-line
-						<a className="thank-you-card__button" onClick={ this.startTransfer }>
-							{ translate( 'Start Domain Transfer' ) }
-						</a>
-					),
-					description: translate( "Now let's get your domain transferred." ),
-				};
-			}
+		} else if ( wasDotcomPlanPurchased && delayedTransferPurchase && this.isNewUser() ) {
```

Comment:
I'd make it even simpler: just check for `delayedTransferPurchase`. `isNewUser` has a weird check to see if the user was created in the last 2 hours - not sure why it's needed here. Definitely should _not_ control if we show this Thank you page or not. And you can purchase a transfer in without a plan, so the plan check does not seem to be needed as well, IMHO.

Reply:
Gotcha! I've updated the logic. :)